2-mandatory/3-groceries.js

```diff
@@ -27,14 +27,34 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+for (let day in weeklyMealPlan) {
+  for (let index = 0; index < day.length; index++) {
+    const element = day[index];
+    if (indexOf(element) === -1) {
```

Comment:
You might have to look back into this function again my friend!! 
IndexOf function is an array method you have to use like this `array.indexOf(item)`
Have you a look at this article to learn more about it if you want
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf